distribution/sdk/sample-nsili-client/src/main/java/org/codice/alliance/nsili/client/NsiliClient.java

```diff
@@ -595,56 +589,54 @@ public void testProductMgr(ORB orb, POA poa, DAG[] dags) throws Exception {
                 NameListHolder locations = new NameListHolder();
                 request.complete(locations);
             } catch (Exception e) {
-                System.out.println("Unable to get product overview: "
-                        + NsilCorbaExceptionUtil.getExceptionDetails(e));
-                e.printStackTrace();
+                LOGGER.error("Unable to get product overview: ",
+                        NsilCorbaExceptionUtil.getExceptionDetails(e));
```

Comment:
This returns a string so I think it should use the parameterized logging like the others.